tests/Unit/CreateUserTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\services\UserService;
+use PHPUnit\Framework\TestCase;
+
+class CreateUserTest extends \Tests\TestCase
+{
+    protected $service;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->service = app()->make(UserService::class);
+    }
+
+    /**
+     * A basic unit test example.
+     *
+     * @return void
+     */
+    public function testCreateUser()
+    {
+        $data = [
+            'name' => 'Steve',
+            'email' => 'coolmail@mail.com',
+            'password' => 'qwerty',
+        ];
+
+        $createdUser = $this->service->createUser($data);
+        $this->assertInstanceOf(UserService::class, new UserService());
```

Comment:
You checked that new object of user Service is object of user service . Crazy. You should assert that '$createdUser' variable, which was returned by 'createUser' method, is instance of User class.